src/providers/limit-provider.js

```diff
@@ -87,4 +96,51 @@ export default class LimitProvider {
     supportsLimits(domainObject) {
         return domainObject.type.startsWith('yamcs.');
     }
+
+    getLimitsForParameter(id, fullId) {
+        let url = `${this.url}api/archive/${this.instance}`;
+        url += '/parameters' + idToQualifiedName(id);
+        url += '?limit=1&order=desc';
+
+        let convertToLimits = (results) => this.convertToLimits(fullId, results);
+
+        return fetch(encodeURI(url))
+            .then(res => res.json())
+            .then(convertToLimits);
+    }
+
+    convertToLimits(id, results) {
+        if (!(results.parameter)) {
+            return [];
+        }
+
+        return this.getLimitFromAlarmRange(id, results.parameter[0].alarmRange);;
+    }
+
+    getLimitFromAlarmRange(id, alarmRange) {
+        let limits = {};
+        alarmRange.forEach(alarm => {
+            limits[alarm.level] = {
+                low: {
+                    color: limitConfig[alarm.level],
```

Comment:
here looks like based `alarm.level` both low and high ll have same color?

Reply:
Yes they have the same color. I prefer to be verbose here though. so I wrote them separately.